### DISCUSSION TITLE
xDrip+ Uploaders message of problem

### DISCUSSION THREAD

**Discussion by SofiyaShishova** [NightscoutFoundation/xDrip#1999] [Q&A]
Folder Uploaders in System status has message: REST-API problem: unable to do REST API Upload: Service Unavailable marking record: failed. xDrip stopped loading data in Nightscout. What is reason? How to fix it? I didn't any change of settings.

**Comment by tolot27** [comment]
It looks like you have enabled API Upload (Rest) in the Cloud-Upload setttings to upload your data to Nightscout. Do you use Nightscout? If not, just deactivate it. Otherwise, verify your Nightscout base URL and API_SECRET.

**Reply by SofiyaShishova** [reply]
Yes, I use Nightscout. It worked well over 2 years before. And stopped yesterday evening. I just checked Basic URL. It isn't changed and correct.

**Reply by SofiyaShishova** [reply]
I use Nightscout as tool for following my data. My father did it for me.

**Comment by tolot27** [comment]
Can you access Nightscout via web interface? What is shown in the logs?

**Reply by SofiyaShishova** [reply]
Yes, I can open Nightscout. No data there since yesterday. Which log do you mean?

**Reply by tolot27** [reply]
The xDrip logs (three vertical points at the top-right of the xDrip home screen).

**Comment by tolot27** [comment]
Also, we don't know your environment and cannot help you much. A good support request contains:

- device name(s) and type(s)
- software versions (Android, xDrip)
- sensors
- share/flower services
- etc.

**Comment by SofiyaShishova** [comment]
I tried to send log from xDrip to my email and didn't get yet. Other way to get log? I use xDrip on my mobile Samsung Galaxy A52. Android ver 12. Sensor is Libre 2. I use last version xDrip.

**Comment by SofiyaShishova** [comment]
No log on my email so far. How to get log? Print screen? Do you need any other info of xDrip settings?

**Reply by tolot27** [reply]
Yes, please create a screenshot and drop it here. That's the easiest way for know.

**Comment by SofiyaShishova** [comment]
![WhatsApp Image 2022-02-17 at 19 05 14](https://user-images.githubusercontent.com/50146352/154521615-fe354db0-751d-43a0-bd78-7d70895a10ee.jpeg)

**Comment by SofiyaShishova** [comment]
It is amazing, but it starts work again! Just now! Thank you very much. Sorry to disturb you.
